src/unknown/disk.rs

```diff
@@ -42,6 +42,10 @@ impl DiskInner {
     pub(crate) fn refresh(&mut self) -> bool {
         true
     }
+
+    pub(crate) fn usage(&self) -> DiskUsage {
+        unreachable!()
```

Comment:
I'd prefer if it returned an empty `DiskUsage` if you don't mind. Like that we can eventually instantiate a `Disk` on all platforms and use its methods without worrying about whether or not it might panic.

Reply:
Makes sense. I will update that along with the `kind` and `name` functions to not panic